GirafIntegrationTest/testlib.py

```diff
@@ -5,6 +5,7 @@
 import warnings
 import io
 from PIL import Image
+from http import HTTPStatus
```

Comment:
Seems a bit odd to import it here to import it elsewhere. Shouldn't it be imported like this in the test files instead?